gst/parser/gstttmlparse.cpp

```diff
@@ -1835,7 +1868,7 @@ static GstStaticCaps ttml_caps = GST_STATIC_CAPS ("application/ttml+xml");
 
 
 static void
-gst_ttmlparse_type_find (GstTypeFind * tf, gpointer private)
+gst_ttmlparse_type_find (GstTypeFind * tf, gpointer priv)
```

Comment:
excessive rename?

Reply:
@roman380  It won't compile because `private` is a key word in C++